templates/chat.mustache

```diff
@@ -0,0 +1,54 @@
+<div id="webchat" role="main"></div>
```

Comment:
Missing header? For example:
```mustache
{{!
    This file is part of Moodle - https://moodle.org/

    Moodle is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

    Moodle is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
}}
{{!
    @template block_iagora/chat

    Display the iagora chat block.

    Classes required for JS:
    * none

    Data attributes required for JS:
    * none

    Context variables required for this template:
    * chatId: The ID of the chat div element where to render the chat block.
    * tokenEndpointURL: The Copilot Token Endpoint URL for native apps.

    Example context (json):
    {
        "chatId": "iagora_chat_55a6cad25a0f6",
        "tokenEndpointURL": "https://insert/your/copilot/endpoint/url",
    }
}}
```

Reply:
Done